src/bors/handlers/review.rs

```diff
@@ -52,6 +55,21 @@ pub(super) async fn command_unapprove(
     notify_of_unapproval(&repo_state, pr).await
 }
 
+pub(super) async fn handle_pull_request_edited(
+    repo_state: Arc<RepositoryState>,
+    db: Arc<PgDbClient>,
+    payload: PullRequestEdited,
+) -> anyhow::Result<()> {
+    // If the base branch has changed, unapprove the PR
```

Comment:
Very nice that we can get this directly from the GH webhook, instead of having to store the original base ref in our DB! Well, we will need to store it anyway, so that we know with which branch to actually merge :laughing: But still.